pkg/v1/mutate/mutate.go

```diff
@@ -268,6 +271,14 @@ func extract(img v1.Image, w io.Writer) error {
 			// name, we may have duplicate entries, which angers tar-split.
 			header.Name = filepath.Clean(header.Name)
 
+			// tar.Next() sometimes mistakenly guesses format as USTAR, which creates a problem:
+			// if the header.Name is > 100 characters long, WriteHeader() returns an error like
+			// "archive/tar: cannot encode header: Format specifies USTAR; and USTAR cannot encode Name=...".
+			// To fix, change format from USTAR to PAX on Windows
+			if isWin && header.Format == tar.FormatUSTAR {
+				header.Format = tar.FormatPAX
+			}
```

Comment:
Ugh. This seems like the kind of thing I'd expect archive/tar to handle for us... :-/ 

Can you add a link to the issue in the comment, and maybe any other helpful issues you've found while debugging this? That will help remind me better why this is here when I rediscover it in six months, or if I have the same Windows-specific issue elsewhere and come back here to try to remember what the deal was.

Reply:
There is also viable path that linuxkit chose - to force PAX on output https://github.com/linuxkit/linuxkit/pull/3141
Maybe force PAX and document it using reasoning in pull request above?

Reply:
something like here (with linked pull request)(https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/moby/image.go#L216)

Reply:
I'd be okay with just always setting this.

Reply:
SGTM 👍